### ISSUE TITLE
Add hint to explanations popups

### ISSUE THREAD

**Issue by ricardoboss** [ricardoboss/Prolangle#92]
In case users don't immediately recognize that some metadata has a popup, we should add a hint. Also, we should refer to the explanations page.

Maybe we can also highlight a metadatum with a popup with an animation if the the user first visits the site.

**Comment by chucker** [comment]
>In case users don't immediately recognize that some metadata has a popup, we should add a hint.

I'm confused. Aren't the dotted underline and the help cursor those hints?

**Comment by ricardoboss** [comment]
Yeah, but they are not enough. Maybe it is also hard to make out on dim/small screens and/or with impaired vision.

**Comment by chucker** [comment]
Sure. I took a stab at this at #96

**Comment by chucker** [comment]
Another aspect: #101

**Comment by ricardoboss** [comment]
I think this issue can be closed with the newly added hints.

**Comment by ricardoboss** [comment]
We may want to add the animation for first-time visitors
